.github/workflows/sync-rest-api-docs.yaml

```
@@ -0,0 +1,49 @@
+name: Sync Camunda REST API
+
+on:
+  schedule:
+    - cron: "0 15 * * 4" # Runs every Thursday at 3 PM UTC
+  workflow_dispatch:
+
+jobs:
+  sync-file:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Camunda docs repository
+        uses: actions/checkout@v3
+        with:
+          repository: camunda/camunda-docs
+          path: docs
+
+      - name: Checkout Camunda repository
+        uses: actions/checkout@v3
+        with:
+          repository: camunda/camunda
+          path: rest-spec
+          sparse-checkout: zeebe/gateway-protocol/src/main/proto/rest-api.yaml
+          sparse-checkout-cone-mode: false
+
+      - name: Sync documentation
+        run: |
+          cd docs
+          mv ../rest-spec/zeebe/gateway-protocol/src/main/proto/rest-api.yaml api/camunda/camunda-openapi.yaml
+          npm i
+          npm run api:generate:camunda
+          if git diff-index --quiet HEAD; then
+            echo "No changes to commit."
+            exit 0
+          fi
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
+        with:
+          token: "${{ secrets.GITHUB_TOKEN }}"
+          path: docs
+          title: Update camunda REST API doc
+          body: This PR updates the camunda REST API
+          commit-message: "docs: update REST API spec"
+          branch: update-rest-doc
+          delete-branch: true
+          reviewers: camunda/tech-writers
+          base: main
```

```suggestion
          base: main
          labels: deploy
```

non-blocking: Do we want to default to deploying a preview site? I have no strong opinion here, but if others do, I think adding this line would do it.

---

Preview site will be the easiest way for us to validate the changes and do any final review. This would be my preference, but like you said, it's non-blocking.

---

I like this 👍